[2024-07-14T04:46:02.802+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-14T04:46:02.984+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ELT-PROJECT.transform.raw_customers.test scheduled__2024-07-06T00:00:00+00:00 [queued]>
[2024-07-14T04:46:03.141+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ELT-PROJECT.transform.raw_customers.test scheduled__2024-07-06T00:00:00+00:00 [queued]>
[2024-07-14T04:46:03.142+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-14T04:46:03.245+0000] {taskinstance.py:2330} INFO - Executing <Task(DbtTestLocalOperator): transform.raw_customers.test> on 2024-07-06 00:00:00+00:00
[2024-07-14T04:46:03.295+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=21420) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-14T04:46:03.299+0000] {standard_task_runner.py:63} INFO - Started process 21441 to run task
[2024-07-14T04:46:03.293+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'ELT-PROJECT', 'transform.raw_customers.test', 'scheduled__2024-07-06T00:00:00+00:00', '--job-id', '1601', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmpfevgthag']
[2024-07-14T04:46:03.306+0000] {standard_task_runner.py:91} INFO - Job 1601: Subtask transform.raw_customers.test
[2024-07-14T04:46:03.368+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-07-14T04:46:03.620+0000] {task_command.py:426} INFO - Running <TaskInstance: ELT-PROJECT.transform.raw_customers.test scheduled__2024-07-06T00:00:00+00:00 [running]> on host airflow-scheduler
[2024-07-14T04:46:04.372+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Data-Ninja' AIRFLOW_CTX_DAG_ID='ELT-PROJECT' AIRFLOW_CTX_TASK_ID='transform.raw_customers.test' AIRFLOW_CTX_EXECUTION_DATE='2024-07-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-06T00:00:00+00:00'
[2024-07-14T04:46:04.389+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-14T04:46:04.393+0000] {local.py:174} INFO - Could not import dbtRunner. Falling back to subprocess for invoking dbt.
[2024-07-14T04:46:04.409+0000] {local.py:292} INFO - Cloning project to writable temp directory /tmp/tmpk50xu7kk from /opt/airflow/dags/dbt/dbt_project
[2024-07-14T04:46:04.453+0000] {local.py:303} INFO - Partial parse is enabled and the latest partial parse file is /tmp/cosmos/ELT-PROJECT__transform/target/partial_parse.msgpack
[2024-07-14T04:46:04.841+0000] {base.py:84} INFO - Using connection ID 'snow_conn' for task execution.
[2024-07-14T04:46:04.848+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-07-14T04:46:04.858+0000] {config.py:298} INFO - Profile caching is enable.
[2024-07-14T04:46:04.867+0000] {config.py:272} INFO - Profile found in cache using profile: /tmp/cosmos/profile/a0017dab0633177292578162811b91dcb9c56d6e54017e9660056cc364a762ad/profiles.yml.
[2024-07-14T04:46:04.875+0000] {local.py:247} INFO - Trying to run the command:
 ['/opt/airflow/dbt_venv/bin/dbt', 'deps', '--project-dir', '/tmp/tmpk50xu7kk', '--profiles-dir', '/tmp/cosmos/profile/a0017dab0633177292578162811b91dcb9c56d6e54017e9660056cc364a762ad', '--profile', 'dbt_project', '--target', 'dev']
From /tmp/tmpk50xu7kk
[2024-07-14T04:46:04.889+0000] {subprocess.py:60} INFO - Tmp dir root location: 
 /tmp
[2024-07-14T04:46:04.931+0000] {subprocess.py:73} INFO - Running command: ['/opt/airflow/dbt_venv/bin/dbt', 'deps', '--project-dir', '/tmp/tmpk50xu7kk', '--profiles-dir', '/tmp/cosmos/profile/a0017dab0633177292578162811b91dcb9c56d6e54017e9660056cc364a762ad', '--profile', 'dbt_project', '--target', 'dev']
[2024-07-14T04:46:05.054+0000] {subprocess.py:84} INFO - Command output:
[2024-07-14T04:46:22.710+0000] {subprocess.py:94} INFO - [0m04:46:22  Running with dbt=1.8.3
[2024-07-14T04:46:24.566+0000] {subprocess.py:94} INFO - [0m04:46:24  Updating lock file in file path: /tmp/tmpk50xu7kk/package-lock.yml
[2024-07-14T04:46:24.618+0000] {subprocess.py:94} INFO - [0m04:46:24  Installing dbt-labs/dbt_utils
[2024-07-14T04:46:25.815+0000] {subprocess.py:94} INFO - [0m04:46:25  Installed from version 0.8.6
[2024-07-14T04:46:25.822+0000] {subprocess.py:94} INFO - [0m04:46:25  Updated version available: 1.2.0
[2024-07-14T04:46:25.834+0000] {subprocess.py:94} INFO - [0m04:46:25
[2024-07-14T04:46:25.839+0000] {subprocess.py:94} INFO - [0m04:46:25  Updates available for packages: ['dbt-labs/dbt_utils']
[2024-07-14T04:46:25.860+0000] {subprocess.py:94} INFO - Update your versions in packages.yml, then run dbt deps
[2024-07-14T04:46:28.638+0000] {subprocess.py:98} INFO - Command exited with return code 0
[2024-07-14T04:46:28.641+0000] {local.py:254} INFO - Update your versions in packages.yml, then run dbt deps
[2024-07-14T04:46:28.643+0000] {local.py:247} INFO - Trying to run the command:
 ['/opt/airflow/dbt_venv/bin/dbt', 'test', '--models', 'raw_customers', '--project-dir', '/tmp/tmpk50xu7kk', '--profiles-dir', '/tmp/cosmos/profile/a0017dab0633177292578162811b91dcb9c56d6e54017e9660056cc364a762ad', '--profile', 'dbt_project', '--target', 'dev']
From /tmp/tmpk50xu7kk
[2024-07-14T04:46:28.646+0000] {subprocess.py:60} INFO - Tmp dir root location: 
 /tmp
[2024-07-14T04:46:28.652+0000] {subprocess.py:73} INFO - Running command: ['/opt/airflow/dbt_venv/bin/dbt', 'test', '--models', 'raw_customers', '--project-dir', '/tmp/tmpk50xu7kk', '--profiles-dir', '/tmp/cosmos/profile/a0017dab0633177292578162811b91dcb9c56d6e54017e9660056cc364a762ad', '--profile', 'dbt_project', '--target', 'dev']
[2024-07-14T04:46:28.832+0000] {subprocess.py:84} INFO - Command output:
[2024-07-14T04:46:45.331+0000] {subprocess.py:94} INFO - [0m04:46:45  Running with dbt=1.8.3
[2024-07-14T04:46:49.812+0000] {subprocess.py:94} INFO - [0m04:46:49  Registered adapter: snowflake=1.8.3
[2024-07-14T04:46:54.733+0000] {subprocess.py:94} INFO - [0m04:46:54  Found 20 models, 27 data tests, 11 sources, 684 macros
[2024-07-14T04:46:54.758+0000] {subprocess.py:94} INFO - [0m04:46:54
[2024-07-14T04:47:05.994+0000] {subprocess.py:94} INFO - [0m04:47:05  Concurrency: 1 threads (target='dev')
[2024-07-14T04:47:05.997+0000] {subprocess.py:94} INFO - [0m04:47:05
[2024-07-14T04:47:06.139+0000] {subprocess.py:94} INFO - [0m04:47:06  1 of 2 START test not_null_raw_customers_COMPANYNAME ........................... [RUN]
[2024-07-14T04:47:08.676+0000] {subprocess.py:94} INFO - [0m04:47:08  1 of 2 PASS not_null_raw_customers_COMPANYNAME ................................. [[32mPASS[0m in 2.53s]
[2024-07-14T04:47:08.686+0000] {subprocess.py:94} INFO - [0m04:47:08  2 of 2 START test not_null_raw_customers_CUSTOMERID ............................ [RUN]
[2024-07-14T04:47:10.501+0000] {subprocess.py:94} INFO - [0m04:47:10  2 of 2 PASS not_null_raw_customers_CUSTOMERID .................................. [[32mPASS[0m in 1.80s]
[2024-07-14T04:47:10.530+0000] {subprocess.py:94} INFO - [0m04:47:10
[2024-07-14T04:47:10.533+0000] {subprocess.py:94} INFO - [0m04:47:10  Finished running 2 data tests in 0 hours 0 minutes and 15.76 seconds (15.76s).
[2024-07-14T04:47:11.615+0000] {subprocess.py:94} INFO - [0m04:47:11
[2024-07-14T04:47:11.618+0000] {subprocess.py:94} INFO - [0m04:47:11  [32mCompleted successfully[0m
[2024-07-14T04:47:11.626+0000] {subprocess.py:94} INFO - [0m04:47:11
[2024-07-14T04:47:11.629+0000] {subprocess.py:94} INFO - [0m04:47:11  Done. PASS=2 WARN=0 ERROR=0 SKIP=0 TOTAL=2
[2024-07-14T04:47:14.831+0000] {subprocess.py:98} INFO - Command exited with return code 0
[2024-07-14T04:47:14.833+0000] {local.py:254} INFO - [0m04:47:11  Done. PASS=2 WARN=0 ERROR=0 SKIP=0 TOTAL=2
[2024-07-14T04:47:16.175+0000] {local.py:196} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/manifest/v12.json is above dbt-ol supported version 7. This might cause errors.
[2024-07-14T04:47:16.194+0000] {local.py:196} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/run-results/v6.json is above dbt-ol supported version 5. This might cause errors.
[2024-07-14T04:47:16.222+0000] {local.py:349} INFO - Inlets: [Dataset(uri='snowflake://OU00370.europe-west2.gcp/ELT_BATCH.raw.raw_customers', extra=None)]
[2024-07-14T04:47:16.250+0000] {local.py:350} INFO - Outlets: []
[2024-07-14T04:47:16.260+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-07-14T04:47:16.463+0000] {dag.py:3954} INFO - Setting next_dagrun for ELT-PROJECT to 2024-07-14 00:00:00+00:00, run_after=2024-07-15 00:00:00+00:00
[2024-07-14T04:47:16.674+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-14T04:47:16.798+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ELT-PROJECT, task_id=transform.raw_customers.test, run_id=scheduled__2024-07-06T00:00:00+00:00, execution_date=20240706T000000, start_date=20240714T044602, end_date=20240714T044716
[2024-07-14T04:47:16.942+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-14T04:47:17.204+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-14T04:47:17.211+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
